ethexe/service/src/lib.rs

```diff
@@ -723,3 +741,61 @@ impl Service {
         Ok(tx_hash)
     }
 }
+
+mod identity {
+    use super::*;
+
+    pub struct ServiceIdentity {
```

Comment:
It doesnt look like service rn. maybe just IdentityStore or smth?

Reply:
Actually here I mean Identity of Service. For services we have other structure: Something**Service**, where Service is in the end.